File: server/clear_database.js
var MongoClient = require('mongodb').MongoClient
  , assert = require('assert');

// Connection URL
var url = 'mongodb://localhost:27017/pairwise-collector';

// Use connect method to connect to the Server
MongoClient.connect(url, function(err, db) {
  assert.equal(null, err);
  console.log("Connected correctly to server");

  var settings = db.collection('settings');
  var users = db.collection('users');
  var data = db.collection('data');

  // Deletes all data from the users and data collections.
  users.remove(function(err, r) {});
  data.remove(function(err, r) {});
});

File: server/clear_database.py
# Clears all data from the database, but leaves the core settings
# record intact.
from pymongo import MongoClient
import sys
import os

client = MongoClient('localhost', 27017)
db = client.attributes

db.users.delete_many({})
db.data.delete_many({})
db.settings.delete_many({"type" : "example"})
db.settings.delete_many({"type" : "attribute"})